### PR TITLE
feat: add label passthrough

### PR DESCRIPTION
**Pull request by maxrimue**
Add label passthrough to check eg whether a given talk is a lightning talk

### CODE REVIEW

File: routes/talks.js [hunk 1]
```diff
@@ -53,12 +53,19 @@ function talks(req, res) {
             description
           } = extractTalk(body)
 
+          let isLightningTalk = false
+          for (const obj of labels) {
+            if (obj.name === 'Lightning Talk') {
+              isLightningTalk = true
+            }
+          }
```
Comment (timche) [inline routes/talks.js]: This or as oneliner `const isLightningTalk = labels.some(label => label.name === 'Lightning Talk')`? 😄 

Also should `Lightning Talk` be filtered out of `labels`?

Comment (maxrimue) [reply routes/talks.js]: `labels` is an array containing objects, so unfortunately `Array.prototype.includes()` won't work here I think. The idea with filtering it out makes sense tho, on it

File: routes/talks.js [hunk 2]
```diff
@@ -52,11 +53,19 @@ function talks(req, res) {
             description
           } = extractTalk(body)
 
+          const lightningTalkLabelName = 'Lightning Talk'
+          const isLightningTalk = labels.some(
+            label => label.name === lightningTalkLabelName
+          )
+
+          labels.filter(obj => obj.name !== lightningTalkLabelName)
```
Comment (timche) [inline routes/talks.js]: [`Array.filter` returns a new array.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)

File: routes/talks.js [hunk 3]
```diff
@@ -52,11 +53,19 @@ function talks(req, res) {
...
+
           return decamelizeKeys({
             title,
             description,
             date,
             updatedAt,
+            isLightningTalk,
```
Comment (timche) [inline routes/talks.js:66]: - `labels` is missing.
- I suggest to move `labels.filter` here as `labels: labels.filter(// ...)`, so you don't have to mutate `labels`.

**Comment by maxrimue** [comment]
Ready for merge cc: @timche

**Review by timche — APPROVED** [review]

**Pull request merged**